apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1091,22 +1091,24 @@ send_response(S, Type, Response) ->
     Msg = aec_peer_messages:serialize_response(Type, Response),
     send_msg(S, response, Msg).
 
-%% If the message is more than 65533 bytes it won't fit in a single Noise
-%% message and we need to fragment it.
+%% If the message is more than 65535 - 2 - 16 bytes (2 bytes for the length and
```

Comment:
Re 16 - I see 16 bytes mentioned in [doc](https://noiseprotocol.org/noise.html#message-format). I see [MAX_AD_LEN](https://github.com/aeternity/enoise/blob/71aac15db7b934e0aecac87ff0175abf40aaa2f2/include/enoise.hrl#L2) though unused.

Reply:
I'll make a note of these and try polish up Enoise as part of https://www.pivotaltracker.com/story/show/157143024